samples/msal-browser-samples/VanillaJSTestApp2.0/app/default/index.html

```diff
@@ -1,70 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
-    <title>Quickstart | MSAL.JS Vanilla JavaScript SPA</title>
-    
-    <script src="../../lib/index.js" type="text/javascript"></script>
-    
-    <!-- adding Bootstrap 4 for UI components  -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-    <link rel="SHORTCUT ICON" href="https://c.s-microsoft.com/favicon.ico?v2" type="image/x-icon">
-  </head>
-  <body>
-    <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
-      <a class="navbar-brand" href="/">MS Identity Platform</a>
-      <div class="btn-group ml-auto dropleft">
-          <button type="button" id="SignIn" class="btn btn-secondary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            Sign In
-          </button>
-          <div class="dropdown-menu">
-            <button class="dropdown-item" id="popup" onclick="signIn(this.id)">Sign in using Popup</button>
-            <button class="dropdown-item" id="redirect" onclick="signIn(this.id)">Sign in using Redirect</button>
-          </div>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+  <title>Quickstart | MSAL.JS Vanilla JavaScript SPA</title>
+
+  <script src="../../lib/msal-browser.js" type="text/javascript"></script>
```

Comment:
Should we refactoring this sample to import the module instead of referencing a script?  nothing to do with the PR really... so approving.